graphics/openvdb/Portfile

```diff
@@ -7,8 +7,7 @@ PortGroup               active_variants 1.1
 PortGroup               compiler_blacklist_versions 1.0
 PortGroup               boost 1.0
 
-github.setup            AcademySoftwareFoundation openvdb 8.0.1 v
-revision                0
+github.setup            AcademySoftwareFoundation openvdb 8.1.0 v
 categories              graphics
```

Comment:
Current practice is to retain the "revision 0".

Reply:
Oops, that was an accident. My local copy of the Portfile blew the line away when I copied it over to my git area.